src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/iOS/PopupRenderer.ios.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System;
+using System.ComponentModel;
+using CoreGraphics;
+using UIKit;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportRenderer(typeof(BasePopup), typeof(PopupRenderer))]
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class PopupRenderer : UIViewController, IVisualElementRenderer
+	{
+		bool isDisposed;
+
+		protected static WeakEventManager EventManager { get; private set; } = new WeakEventManager();
```

Comment:
@pictos I updated the WeakEventManager to be a local static so it can be easily accessed between the `PopupRenderer` and `PopupDelegate`. Does this solve the Garbage Collection issue we were discussing earlier?